# GPS data includes two latitude readings, two longitude readings, and two sspeed over ground readings

float32 heading
float32 roll
float32 pitch
float32 lin_x
float32 lin_y
float32 lin_z
